outlines/welcome.md

```diff
@@ -0,0 +1,13 @@
+1. Welcome to the Meteor guide
+  1. This is a set of articles outlining opinions on best-practice application development in the Meteor framework
+  2. Nothing here is prescriptive (i.e. you can use the platform in many other ways), but this is how we think apps are best developed.
```

Comment:
I think the Guide _is_ prescriptive. The framework is flexible. But we can say that you can take as many or as few of the opinions as you want.